appserver/node-express/lib/db-client/qnaDoc/patch.js

```diff
@@ -66,6 +116,24 @@ module.exports = function (txid, spec) {
     );
   };
 
+  /**
+   * Handle the post of a new answer to a Samplestack question. The posted
+   * answer is merged with an answer template object, a unique ID, contributor
+   * information, and the current time. The resulting object is written to the
+   * MarkLogic database. During the write, the answer count for the question is
+   * incremented and the last activity date is updated.
+   *
+   * @param  {String} txid The transaction ID.
+   * @param  {Object} contributor Contributor object. Example:
+   *   {"id":"cf99542d-f024-4478-a6dc-7e723a51b040",
+   *    "displayName":"JoeUser"}
+   * @param  {String} questionId The question ID.
+   * @param  {Object} spec The answer content from the request. Example:
+   *   {"comments":[],
```

Comment:
Great, thanks for the clarification. The comments are coming in from the browser (your first bullet above).  I will have the middle tier dutifully store it (which means making no changes in the middle tier; it's doing what it needs to do now). 

Now that I revisited how our "draft" answer works, I understand why that empty comments array is there.